Python/Product/Workspace/PythonLaunchDebugTargetProvider.cs

```diff
@@ -134,7 +134,7 @@ class PythonLaunchDebugTargetProvider : ILaunchDebugTargetProvider {
             var launchConfig = new LaunchConfiguration(config) {
                 InterpreterPath = config == null ? path : null,
                 InterpreterArguments = settings.GetValue(InterpreterArgumentsKey, string.Empty),
-                ScriptName = scriptName,
+                ScriptPath = Path.Combine(workingDir, scriptName),
```

Comment:
We should only do this if it's not already a full path.
You can check with `Path.IsPathRooted`

Reply:
Fixed